platform_versioned_docs/version-24.1/data_studios/managing.mdx

```diff
@@ -0,0 +1,152 @@
+
+## Connect to a data studio
+
+This functionality is available to all user roles excluding the **View** role.
+
+To connect to a running session, select the three dots next to the status message and choose **Connect**. A new browser tab will open containing the data studio session.
+
+:::warning
+An active connection to a session will not prevent administrative actions that might disrupt that connection. For example, a session can be stopped by another workspace user while you are active in the session, the underlying credentials can be changed, or the compute environment can be deleted. These are independent actions and the user in the session won't be alerted to any changes - the only alert will be a server connection error in the active session browser tab.
+:::
+
+Once connected, the data studio session list will update to display the status of **running** and any connected user will have their avatar displayed under the status in both the list of Data Studios and in each data studio session detail page.
+
+## Collaborate in a data studio
+
+This functionality is available to all user roles excluding the **View** role.
+
+To share a link to a running session with collaborators inside your workspace, select the three dots next to the status message for the session you want to share, then select **Copy data studio URL**. Using this link, other authenticated users can access the session directly.
+
+![](../data/_images/data_studios_collaboration.png)
+
+:::note
+Collaborators need valid workspace permissions to connect to the running data studio.
+:::
+
+## Stop a data studio
+
+This functionality is available to all user roles with the **Maintain** role and above.
+
+To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a session is stopped, the compute resources it's using are deallocated. You can stop a session at any time, except when it is **starting**.
+
+Stopping a running session creates a new checkpoint.
+
+## Restart a stopped data studio
+
+This functionality is available to all user roles with the **Maintain** role and above.
+
+When you restart a stopped session, the session uses the most recent checkpoint.
+
+## Start a new session from a checkpoint
+
+This functionality is available to all user roles with the **Maintain** role and above.
+
+You can start a new session from an existing stopped session. This will inherit the history of the parent checkpoint state. From the list of **stopped** data studios in your workspace, select the three dots next to the status message for the data studio and select **Start as new**. Alternatively, from the Data Studio detail page select the **Checkpoints** tab and select the three dots in the **Actions** column then select **Start as new**to start a new session.
```

Comment:
```suggestion
You can start a new session from an existing stopped session. This will inherit the history of the parent checkpoint state. From the list of **stopped** data studios in your workspace, select the three dots next to the status message for the data studio and select **Start as new**. Alternatively, select the **Checkpoints** tab on the data studio detail page, select the three dots in the **Actions** column, and then select **Start as new** to start a new session.
```